pkg/client/helper.go

```diff
@@ -62,8 +63,20 @@ type Config struct {
 
 	// Server requires TLS client certificate authentication
 	CertFile string
-	KeyFile  string
-	CAFile   string
+	// Server requires TLS client certificate authentication
+	KeyFile string
+	// Trusted root certificates for server
+	CAFile string
```

Comment:
Ideally we would have a set of CAs that we trust.  As this stuff is new, it might be easier to make this be a list now instead of fixing it up later.

If/when a user needs to rotate CA certs, having a list here will make it *much* easier.